pkgs/by-name/ad/ada-url/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  cpm-cmake,
+  simdjson,
+  cmake,
+  cxxopts,
+}:
```

Comment:
You may want to follow the formatting `nixpkgs-fmt pkgs/by-name/ad/ada-url/package.nix`

Reply:
I took a look at it, and it seems that there is an ambiguous situation:

The [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/264f5eb58cadb4592f588f1d7a77c060a0769297/CONTRIBUTING.md?plain=1#L675) showcase the compact style:

```nix
{ lib
, fetchFromGitHub
, stdenv
, cpm-cmake
, simdjson
, cmake
, cxxopts,
}:
```
while the RFC 0166 formatter rewrite it the following way
```nix
{
  lib,
  fetchFromGitHub,
  stdenv,
  cpm-cmake,
  simdjson,
  cmake,
  cxxopts,
}:
```

I will seek for some help to clarify which is best to use

Reply:
Seems like *both are (currently) correct* from NobbZ.